modules/bootstrap/iam_users/admin_users.tf

```diff
@@ -3,6 +3,13 @@ resource "aws_iam_user" "admin" {
   name     = each.value
 }
 
+# give each user a temporary password for logging into aws web console
+resource "aws_iam_user_login_profile" "admin" {
+  for_each                = toset([for user in aws_iam_user.admin : user.name])
+  user                    = each.value
+  password_reset_required = true
```

Comment:
Will this run a password reset each time `tofu apply` is run?

Reply:
Nope, only the first time.

Reply:
It just did it to you and I because our accounts existed _without_ the login profile.

Reply:
:sunglasses: